src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -603,7 +603,7 @@ protected void InjectDeployments(ActorSystem system, RoleName role)
                         // controller node is finished/exited before r.addr is run
                         // on the other nodes
                         var unresolved = "akka://unresolved-replacement-" + r.Role.Name;
-                        Log.Warning(unresolved + " due to: " + e.ToString());
+		         Log.Warning(unresolved + " due to: {0}", e.ToString());
```

Comment:
Could you remove the .ToString() part on the `e`? normal loggers will do the tostringing by themselves and serilog needs the entire object to write the exception details